### ISSUE TITLE
Contains(Vector3[] points) is not implemented in BoundingFrustum

### ISSUE THREAD

**Issue by QuantumDeveloper** [sharpdx/SharpDX#749]
Methos Contains(Vector3[] points) not implemented and because of this operation like ``frustum.Contains(anotherFrustum)`` cannot be performed.

Is it possible to fix (implement) that method?

**Comment by xoofx** [comment]
Yes, though I have personally no more spare time left for SharpDX, apart for merging PR and integrate bug fixes from the community.

**Comment by amerkoleci** [comment]
I would suggest taking implementation from MonoGame https://github.com/mono/MonoGame/blob/8a8f21c65c6e40688ed75b64c65bbf5a5790e40f/MonoGame.Framework/BoundingFrustum.cs
